### PR TITLE
Added missing migration 0002

### PR DESCRIPTION
**Pull request by jose-lpa**
Some changes were introduced, but they didn't had their correspondent migration: The `user_email` model field is now 254 characters long, unless specified in the `0001_initial` migration. This is because `EmailField` in Django base code is now updated to 245 chars.

This migration fixes that.

### CODE REVIEW

**Comment by claudep** [comment]
What changed for `is_removed`?
As for `user_email`, I'd also set an explicit max_length to the model field, otherwise users of Django 1.7 will have a warning (as EmailField is still 75 chars in 1.7).
Could you also give an explicit name to the migration file?

**Comment by jose-lpa** [comment]
> What changed for is_removed?

Good catch. It was a mistake introduced in [this commit](https://github.com/django/django-contrib-comments/commit/5fade4566c673bb94bb2c7d026d380300fe98cc9#diff-06d47608a9a2f77dd0d352e29b78f52aR33). You can appreciate in the diff how the phrase "*A "This comment has been removed" message will be displayed instead.*" was indented, missing the single space between words *"removed"* and *"message"*. This made Django create a migration for the field.

> As for user_email, I'd also set an explicit max_length to the model field

Done.

> Could you also give an explicit name to the migration file?

Done.

File: django_comments/models.py [hunk 1]
```diff
@@ -59,7 +59,8 @@ class Comment(BaseCommentAbstractModel):
     user = models.ForeignKey(settings.AUTH_USER_MODEL, verbose_name=_('user'),
                              blank=True, null=True, related_name="%(class)s_comments")
     user_name = models.CharField(_("user's name"), max_length=50, blank=True)
-    user_email = models.EmailField(_("user's email address"), blank=True)
+    user_email = models.EmailField(_("user's email address"), max_length=254,
```
Comment (claudep) [inline django_comments/models.py]: Looks good. Maybe add a short comment explaining the reason of the explicit max_length. After that, please rebase and merge your commits.

**Comment by jose-lpa** [comment]
Done. I added a little comment as explanation as you suggested. This PR is already up to date with the master code, so it can be directly merged, no rebases needed.

**Comment by claudep** [comment]
Thanks, pushed in 5d565503b26f7

**Pull request closed without merging**